### PR TITLE
create landing page

### PR DESCRIPTION
**Pull request by Beker94**

### CODE REVIEW

File: src/pages/landing-page/components/find-out/find-out.component.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import img1 from '../../../../images/Landing 3.png';
+import dog from './bigl.svg';
+import styles from './find-out.module.scss';
+
+export const FindOut: React.FC<{}> = () => {
```
Comment (enheit) [inline src/pages/landing-page/components/find-out/find-out.component.tsx]: `React.FC<{}>` -> `React.FC`

File: src/pages/landing-page/components/find-out/find-out.component.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            <td>3-5</td>
+          </tr>
+          <tr>
+            <td className={styles.table__row1}>Кількість прийомів їжі за день</td>
+            <td className={(styles.table__data, styles.table__data_bad)}>5</td>
+            <td>2-4</td>
+          </tr>
+        </tbody>
+      </table>
+      <Link to={`/`} className={styles.block__link}>
```
Comment (enheit) [inline src/pages/landing-page/components/find-out/find-out.component.tsx]: `to={"/"}` -> `to={ROUTES.home}` or `to={ROUTES.root}`

File: src/pages/landing-page/components/find-out/find-out.module.scss [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+        border-radius: 10px;
+    }
+}
+
+
+
+.head {
+    border-radius: 10px;
+
+}
```
Comment (enheit) [inline src/pages/landing-page/components/find-out/find-out.module.scss]: Wrong alignment (use 2 spaces)

File: src/pages/landing-page/components/find-out/find-out.module.scss [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        color: #D42323;
+        border-radius: 10px;
+    }
+}
+
+
+
+.head {
+    border-radius: 10px;
+
```
Comment (enheit) [inline src/pages/landing-page/components/find-out/find-out.module.scss]: Blank line

File: src/pages/landing-page/components/to-pick-up/to-pick-up.component.tsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+import dialogs from '../../../../images/dialog.svg';
+import styles from './to-pick-up.module.scss';
+
+export const ToPickUp: React.FC<{}> = () => {
+  return (
+    <div className={styles.block}>
+      <h2 className={styles.block__title}>Підберіть свого ідеального собаку</h2>
+      <img src={img2} width="200" className={styles.block__img}></img>
+      <img src={dialogs} className={styles.block__img}></img>
+      <Link to={`/`} className={styles.block__link}>
```
Comment (enheit) [inline src/pages/landing-page/components/to-pick-up/to-pick-up.component.tsx]: to={"/"} -> to={ROUTES.home} or to={ROUTES.root}

File: src/pages/landing-page/landing-page.component.tsx [hunk 6]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import styles from './landing-page.module.scss';
+import logo from '../../images/paw-solid.svg';
+
+import { FindOut } from './components/find-out/find-out.component';
+import { ToPickUp } from './components/to-pick-up/to-pick-up.component';
+
+export const LandinPage: React.FC<{}> = () => {
```
Comment (enheit) [inline src/pages/landing-page/landing-page.component.tsx]: `React.FC<{}>` -> `React.FC`

**Review by Pashkaua — APPROVED** [review]
Nice.

**Review by Danik1702 — APPROVED** [review]
It looks pretty well!!

**Review by okhabr — APPROVED** [review]

**Pull request merged**